src/model/entities/action.ts

```diff
@@ -46,6 +46,7 @@ export interface MutateActionWithPlannedPieceMethods {
 }
 
 /**
+ * @remarks
```

Comment:
Why?

Reply:
To make the comment a TSDoc comment.
Knew you'd comment on this.
I can remove it again, if you'd like, but as the comment is a remark, it makes sense to marks it as such, for TSDoc.

Reply:
Well, this isn't "just a remark" for the type, so my opinion would be to not include it.
But it's a decision up to the team whether or not to use it.

Reply:
So untill the team has decided to use it, it should be removed.

Reply:
Removed the `Remark`.